resources/dicts/events/death/general.json

```diff
@@ -1116,7 +1116,7 @@
         ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c has always wondered where {PRONOUN/m_c/subject} came from. {PRONOUN/m_c/subject/CAP} {VERB/m_c/sneak/sneaks} out of camp, intending to find loners who might have some clues about {PRONOUN/m_c/poss} origins. Frantic with worry and confusion after finding m_c's empty nest in the morning, c_n launches a search. They find {PRONOUN/m_c/poss} cooling body after a long search. They wonder why {PRONOUN/m_c/subject} decided to leave the safety of c_n, and what killed {PRONOUN/m_c/object}.",
+        "event_text": "m_c sneaked out of camp, intending to find more about {PRONOUN/m_c/poss} origins. c_n found {PRONOUN/m_c/poss} cooling body in the morning, after a long search.",
```

Comment:
snuck

Reply:
both sneaked and snuck are correct actually!

Reply:
I despise the word sneaked but also I keep hearing it's more grammatically correct (and pycharm flags it as a typo) lmao might be worth a larger style discussion if we want to be consistent with the term clangen uses?

Reply:
I think this is a learned/learnt problem aka it's not our problem that english is Like This and we can go with either whenever